demo/compositions/useFlowMock.ts

```diff
@@ -1,29 +1,62 @@
 import { useSeeds } from './useSeeds'
 import { Flow } from '@/models'
 import { mocker } from '@/services'
-import { repeat } from '@/utilities'
+import { choice, repeat } from '@/utilities'
+
+export type FlowMockOptions = {
+  deploymentsCount: number,
+}
+
+export function useFlowMock(override?: Partial<Flow>, options?: Partial<FlowMockOptions>): Flow {
+  const {
+    deploymentsCount = mocker.create('number', [0, 8]),
```

Comment:
These changes give us a little more control of how downstream objects are created when we're mocking flows